packages/kit/src/runtime/server/page/utils.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param {any} v
+ */
+export function is_pojo(v) {
```

Comment:
It irks me a little that there's another function with the same name which does something different, and both have no short description so you have to jump into the code and usage locations to know what's this about. 
Also: What's the `TODO` about?